src/tests/conformance/post_handshake/cmd/transaction.rs

```diff
@@ -86,5 +85,5 @@ async fn c012_TXN_submit_txn_and_expect_to_receive_it() {
     synthetic_node_rx.shut_down().await;
     synthetic_node_tx.shut_down().await;
     kmd.stop().expect("unable to stop the kmd instance");
```

Comment:
We can update `kmd` also with (these already exist in the const.rs):
```
/// Error message when building a kmd instance fails.
pub const ERR_KMD_BUILD: &str = "unable to build the kmd instance";

/// Error message when a kmd instance fails to stop.
pub const ERR_KMD_STOP: &str = "unable to stop the kmd instance";
```

Reply:
Done.